packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -2220,6 +2221,9 @@ export function attach(
     let displayName = componentInfo.name || '';
     if (typeof env === 'string') {
       // We model environment as an HoC name for now.
+      if (secondaryEnv !== null) {
+        displayName = secondaryEnv + '(' + displayName + ')';
```

Comment:
It is enough to check ` if (secondaryEnv)` , it will make sure it will not be null and not undefined